### PR TITLE
Sublayers fix

### PR DESCRIPTION
**Pull request by lowriech**
This has a couple things, particularly:
- using a custom `pmtile` server to run global vector floodmaps and mangroves
- small updates to `sublayers` to make the data model a little better.

A couple things:
- we could use a test on the `sources`, particularly our custom `pmtile` server.  If you specify a layer that doesn't exist, you don't get any error.  A test would check the server and see what layers are present.
- sublayers management, not complete.  It would be nice to abstract this out so we don't have to individually add logic for it in each layer.

### CODE REVIEW

File: src/hooks/layers/useLayers.js
Comment (gislawill) [inline src/hooks/layers/useLayers.js:1]: Can we remove these console logs to cut down on console noise?

File: src/layers/protos/FillWithOutlineProto.js [hunk 1]
```diff
@@ -33,6 +35,16 @@ export default class FillWithOutlineProto {
...
     this.maxzoom = maxzoom;
+    console.log(id, subgroup);
+    console.log(id, floodGroup, subgroup);
+    console.log(subgroup !== undefined);
+    console.log(subgroup !== undefined && subgroup === floodGroup);
+    this.visible =
+      (subgroup !== undefined && subgroup === floodGroup) ||
+      subgroup === undefined
+        ? "visible"
+        : "none";
```
Comment (gislawill) [inline src/layers/protos/FillWithOutlineProto.js:46]: Nit: Could you remove these consoles logs? I think this logic could also be simplified to
```suggestion
    this.visible = !subgroup || subgroup === floodGroup ? "visible" : "none";
```

File: src/layers/sources.js [hunk 2]
```diff
@@ -39,33 +39,78 @@ const mapbox_dem = {
...
-  'tileSize': 256,
+  tileSize: 256,
 };
 
+const flooding_1996_pt = {
+  id: "flooding_1996_pt",
+  type: "vector",
+  format: "mvt",
+  tiles: [
+    "https://pmtileserver-staging-myzvqet7ua-uw.a.run.app/cwon_data_chunked/with_1996_TC_Tr_100/{z}/{x}/{y}.mvt",
```
Comment (gislawill) [inline src/layers/sources.js:66]: Do you know if this url will change with new deployments? Specifically, can we safely point to a url with "myzvqet7ua" in it long term?

Comment (lowriech) [reply src/layers/sources.js:66]: Yes we can point to this URL stably, but we can also put it behind `api.coastalresilienceexplorer.org`

**Pull request merged**